cmake/Corrosion.cmake

```diff
@@ -321,7 +358,7 @@ function(_corrosion_copy_byproduct_deferred target_name output_dir_prop_name car
     )
 endfunction()
 
-function(_corrosion_call_copy_byproduct_deferred target_name output_dir_prop_name cargo_build_dir file_names)
+function(_corrosion_call_copy_byproduct_deferred target_name output_dir_prop_name cargo_build_dir file_names is_binary)
```

Comment:
The extra parameter is rather ugly. I'm wondering if it's worth porting this over to `cmake_parse_arguments`. But I guess latest in this function it would all have to be unpacked and packed again, so maybe not worth it?

Reply:
hmm, maybe we could just remove all named parameters to this function and pass [[${ARGN}]] to the deferred function.

Reply:
Good idea, I’ll try that.